sycl/test-e2e/ESIMD/PerformanceTests/invoke_simd_smoke.cpp

```diff
@@ -0,0 +1,16 @@
+//==------------- invoke_simd_smoke.cpp  - DPC++ ESIMD on-device test----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu
```

Comment:
we probably want to restrict to a single GPU like we did for the rest of the perf tests

```
// REQUIRES: gpu-intel-dg2 && level_zero
```

Reply:
yep, exactly